v3/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## Version 3.0.0-pre.1389 - October 24, 2023
+
+### Features/Improvements
+- Close component should be undoable as a single action [#186075314](https://www.pivotaltracker.com/story/show/186075314)
+
+### Bug Fixes
+- Wrong attribute sometimes passed to formula in **Plotted Value** [#186266057](https://www.pivotaltracker.com/story/show/186266057)
```

Comment:
Seems like we may be missing some features/improvements/bug fixes. Just looking at the recently merged PRs: #946, #949, #951, #952, #953, #956 could all warrant a mention.

Reply:
Thank you for catching that @kswenson . I've added these now (#949 was already there; I've the remaining 5 and compared the git log output against the changelog list to make sure I haven't missed anything else).